design/fake-client.rst

```diff
@@ -63,16 +67,22 @@ Design
 In the following description, the second design option is used.
 
 There is a fake session object that the user of the zhmcclient package
-will use instead of zhmcclient.Session. The fake session object contains a
+will use instead of `zhmcclient.Session`. The fake session object contains a
 faked HMC that can represent all resources relevant for the zhmcclient.
-There fake session object implements the same API as zhmcclient.Session, so
-that the rest of the zhmcclient classes does not need to be changed.
-The get(), post() and delete() methods of the fake session class will
+
+This fake session object implements the same API as `zhmcclient.Session`, so
+that the rest of the zhmcclient classes do not need to be changed.
+The `get()`, `post()` and `delete()` methods of the fake session class will
 be redirected to operate on the faked HMC.
 
-The fake session object will have the ability to be "loaded" with an
-initial state of resources. The resource definitions can be provided as
-a dictionary of nested dictionaries, following this example::
+The fake session object will have the ability to be populated with a resource
+state. This is supported in two ways that can both be used:
```

Comment:
Supporting two different ways sounds like a lot more effort, but I think I need to see how this plays out in code.